specification/azsadmin/resource-manager/subscriptions/readme.md

```diff
@@ -58,6 +58,12 @@ directive:
       - $.definitions.Subscription
       - $.definitions.PlanAcquisition
       - $.definitions.Location
+
+  - suppress: OperationIdNounVerb
+    reason: Subscriptions is both the name of the module and the operation action name.
+    where:
+      - $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.Subscriptions.Admin/moveSubscriptions"].post.operationId
+      - $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.Subscriptions.Admin/validateMoveSubscriptions"].post.operationId
```

Comment:
we could avoid these suppression by simplifying the method name, like `Subscriptions_Move`, `Subscriptions_ValidateMove`. The generated class looks like

```C#
class Subscriptions
{
    Get(args);
    List();
    Move(args);
    ValidateMove(args);
}
```

Usage would be:

```
Subscriptions subscriptions = new Subscriptions(auth);
subscriptions.move(args);
```

Its a pattern that in general we try not to repeat the resource name in the method names. 
For example, `VirtualMachines.Capture(args)` instead of `VirtualMachines.CaptureVirtualMachine(args)`

Reply:
This simply swaps the warning to failing to include action name in operation id, which is a worse choice.

Reply:
sorry, ignore this comment.

Reply:
you are right, i just talk to linter team - yes action url segment should match the operation name.

Reply:
Opened an issue in validator repo https://github.com/Azure/azure-openapi-validator/issues/171 as these warnings are false positive.